src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs

```diff
@@ -1,6 +1,6 @@
 ﻿//-----------------------------------------------------------------------
 // <copyright company="Microsoft">
-//    Copyright © Microsoft Corporation.  All rights reserved.
+//    Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
Extra spaces.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

Below I see many the same patterns.